contracts/src/DefaultPool.sol

```diff
@@ -37,10 +39,12 @@ contract DefaultPool is Initializable, IDefaultPool {
         collToken = _addressesRegistry.collToken();
         troveManagerAddress = address(_addressesRegistry.troveManager());
         activePoolAddress = address(_addressesRegistry.activePool());
+        activePoolVaultAddress = address(_addressesRegistry.collateralVault());
```

Comment:
Caching activePoolVaultAddress in multiple contracts requires multiple setters. I recommend fetching the address directly from activePool to avoid this redundancy. Also, the naming for the external vault is inconsistent across contracts